dcos/README.md

```diff
@@ -14,6 +14,12 @@ dcos marathon app add webapp.json
 Note the `linkerd-dcos.json` files assume 4 nodes. Modify this to equal the
 total number of public+private nodes in your cluster.
 
+Note that the `linkerd-dcos.json` is one of many ways you can setup
+Linkerd to run in DC\OS. Unfortunately using `linkerd-dcos.json` might make it
+difficult to send operating system signals to Linkerd e.g. `SIGTERM` for
+graceful shutdown. An [alternative](/linkerd-dcos-with-fetch.json) is one way to
```

Comment:
i think you want `[alternative](linkerd-dcos-with-fetch.json)` or `[alternative](/dcos/linkerd-dcos-with-fetch.json)`.

also you may want to note that fetch was introduced in 1.10, and that DC/OS <= 1.9, uses a top-level `uris` field.